packages/fx-core/resource/package.nls.json

```diff
@@ -600,6 +600,7 @@
   "error.teamsApp.validate.details": "File path: %s, title: %s",
   "error.teamsApp.AppIdNotExistError": "Teams app with ID %s does not exist in Teams Developer Portal.",
   "error.teamsApp.InvalidAppIdError": "Teams app ID %s is invalid, must be a GUID.",
+  "error.teamsApp.createAppPackage.invalidFile": "%s is invalid, it should be in the same or subdirectory of manifest.json",
```

Comment:
the same or subdirectory of manifest.json. -> the same directory as manifest.json or a subdirectory of it.